Mathlib/Tactic/Linter/DirectoryDependency.lean

```diff
@@ -220,6 +220,43 @@ def allowedImportDirs : NamePrefixRel := .ofArray #[
   (`Mathlib.Tactic.Linter, `ImportGraph),
   (`Mathlib.Tactic.Linter, `Mathlib.Tactic.MinImports),
   (`Mathlib.Tactic.Linter.TextBased, `Mathlib.Data.Nat.Notation),
+
+  (`Mathlib.Logic, `Batteries),
+  -- XXX: should this import be allowed on a more fine-grained level?
+  (`Mathlib.Logic, `Mathlib.Control),
+  (`Mathlib.Logic, `Mathlib.Lean),
+  (`Mathlib.Logic, `Mathlib.Util),
+  (`Mathlib.Logic, `Mathlib.Tactic),
+  (`Mathlib.Logic.Fin.Rotate, `Mathlib.Algebra.Group.Fin.Basic),
+  (`Mathlib.Logic.Hydra, `Mathlib.GroupTheory),
+  (`Mathlib.Logic, `Mathlib.Algebra.Notation),
+  (`Mathlib.Logic, `Mathlib.Algebra.NeZero),
+  -- XXX: should this import be allowed only on a more fine-grained level (or at all)?
+  (`Mathlib.Logic, `Mathlib.Data),
+  -- XXX: should this import be allowed only on a more fine-grained level (or at all)?
+  (`Mathlib.Logic, `Mathlib.Order),
```

Comment:
I don't know if this one is very bad either, but having `Mathlib/Logic/Function/Basic.lean` import `Mathlib.Order.Defs.Unbundled` is a good idea. Let's turn this into a TODO comment.

```suggestion
  -- TODO: this next dependency should be made more fine-grained.
  (`Mathlib.Logic, `Mathlib.Order),
```